.github/workflows/ci-clarinet-cli.yaml

```diff
@@ -85,46 +85,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Install Rust toolchain
         run: |
-          rustup set profile minimal
-          rustup toolchain install stable --component llvm-tools-preview
```

Comment:
I'm guessing `llvm-tools-preview` is not needed anymore? Was this ever needed?

Reply:
It might have been in the past, I don't know if `cargo test` / `grcov still` needs it, but it's not needed with `nextest`